Ports/qemu/package.sh

```diff
@@ -12,9 +12,11 @@ configopts=(
     '--enable-sdl'
     '--enable-tcg'
     '--enable-tools'
+    '--enable-slirp'
```

Comment:
I hate to do this but these are in A L P H A B E T I C A L order :hyperlinus:

Reply:
Nothing escapes linus' laser eyes!